TrafficCapture/dockerSolution/src/main/docker/migrationConsole/workflows/createSnapshot.yaml

```diff
@@ -0,0 +1,84 @@
+apiVersion: argoproj.io/v1alpha1
+kind: WorkflowTemplate
+metadata:
+  name: create-snapshot
+  # add labels later w/ patch
+spec:
+  entrypoint: snapshot-workflow
+  templates:
+    - name: snapshot-workflow
+      inputs:
+        parameters:
+          - name: snapshot-name
+            value: "migration-assistant-snapshot"
+          - name: source-config
+            value: "{ \"endpoint\": \"http://ma-capture-proxy:9200\" }"
```

Comment:
Hmm, won't this cause a snapshot to be created during replay?  We could avoid this by hitting the target directly, but I'm not sure that is correct either.  This might be out of scope for this change.